Bottom: 616c774ad30056b4af7e3ee897c245f10e574d1d
Top:    f6379ea82a682bfd963ceee857f67ac6800293f0
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-17 13:11:40 +1300

Refresh of double-streamresult

---

diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index dceda9e..3afd9bf 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -570,11 +570,11 @@ class TestDoubleStreamResultEvents(TestCase):
         result = LoggingStreamResult()
         result.startTestRun()
         now = datetime.datetime.now(utc)
-        result.status("foo", "finished", test_tags=set(['tag']),
+        result.status("foo", "success", test_tags=set(['tag']),
             runnable=False, route_code='abc', timestamp=now)
         self.assertEqual(
             [('startTestRun',),
-             ('status', 'foo', 'finished', set(['tag']), False, 'abc', now)],
+             ('status', 'foo', 'success', set(['tag']), False, 'abc', now)],
             result._events)
